tests/test_rdf.py

```diff
@@ -211,6 +211,48 @@ def test_add_rdf(local_model):
     assert named_attributes[0]['value'] == '2.5' or named_attributes[1]['value'] == '2.5'
 
 
-def test_get_unique_cmeta_id(simple_ode_model):
-    assert simple_ode_model.get_unique_cmeta_id('already_unique_id') == 'already_unique_id'
-    assert simple_ode_model.get_unique_cmeta_id('time') == 'time_'
+def test_add_cmeta_id():
+    """ Tests Model.add_cmeta_id(). """
+
+    model = shared.load_model('test_simple_odes')
+
+    # Test on variable without a cmeta id
+    v = model.get_symbol_by_name('circle_x$x')
+    assert v.rdf_identity is None
+    model.add_cmeta_id(v)
+    assert v.rdf_identity is not None
+
+    # Test on variable with a cmeta id
+    v = model.get_symbol_by_ontology_term((shared.OXMETA, 'time'))
+    rid = v.rdf_identity
+    assert rid is not None
+    model.add_cmeta_id(v)
+    assert v.rdf_identity == rid
+
+    # Test on variable with a name that's already in use as a cmeta id
+    assert model.has_cmeta_id('time')
+    v = model.add_variable('time', 'second')
+    model.add_cmeta_id(v)
+
```

Comment:
This bit doesn't actually test anything!

Reply:
It tests that you can call add_cmeta_id on a variable that already has a cmeta id without getting an exception

Reply:
Indeed, but it doesn't check that the rdf_identity is the same afterwards, and probably should?

Reply:
The problem is it's called on a variable without a cmeta id, but one for which the normal strategy of using the variable name fails.

This strategy isn't really part of the public API though! How the clash is resolved definitely isn't, so shouldn't be checking that here?

Reply:
Sorry my comment above was wrong!

Reply:
Ah, good point. So don't check the identity is unchanged, but perhaps do check that it's not None and not the same as for the existing time variable?

Reply:
Good idea!